src/App.js

```diff
@@ -1,31 +1,32 @@
 import "./index.css";
+import headerLogo from "./images/logo_place_header.svg";
+import headerImage from "./images/two_again.png";
 
 const App = () => {
   return (
     <div className="app">
       <header className="header">
         <img
           className="header__logo"
-          src="/src/images/logo_place_header.svg"
+          src={headerLogo}
           width="228px"
           height="32px"
           alt="logo"
         />
-        <div classnames="title">Научиться учиться</div>
-        <div classnames="subtitle">
+        <h1 classnames="title">Научиться учиться</h1>
```

Comment:
Avoid using `div`s
Check the [video](https://www.youtube.com/watch?v=mAoxm-AyNCw) of my friend about it with an examples